packages/lib/src/components/Dropin/components/PaymentMethod/PaymentMethodItem.scss

```diff
@@ -55,15 +55,15 @@
     justify-content: space-between;
     font-weight: 400;
     font-size: $font-size-medium;
-    padding: 16px;
-    padding-left: 48px;
+    padding: $spacing-medium - $spacing-xsmall;
+    padding-left: $spacing-xxlarge - $spacing-xsmall;
     position: relative;
     transition: background 0.1s ease-out;
     width: 100%;
 
     [dir='rtl'] & {
-        padding: 16px;
-        padding-right: 48px;
+        padding: $spacing-medium - $spacing-xsmall;
+        padding-right: $spacing-xxlarge - $spacing-xsmall;
```

Comment:
Can you elaborate why do we need this calculation here to set `44px`?

Reply:
We don't need a calculation, but 44px is a bit of a magic number, this hopefully gives a bit more reasoning on why. 
In this case because the spacing was moved from the padding of the container, to the padding of the button itself.

Reply:
Also, if we all agree this is ugly I will just change it to 44px and 12px. And maybe add a comment somewhere, the objective is readability.